src/main/java/org/cp/confisual/plantuml/NevisAuthObject.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Author : AdNovum Informatik AG
+ */
+
+package org.cp.confisual.plantuml;
+
+public interface NevisAuthObject {
```

Comment:
Rename interface to 'NevisAuthConfigObject', move it to nevisauth package.
Could use more generic name: 'VisitableConfigObject' and add a type parameter for the Visitor class that it wants to accept.